lms/static/js/ccx/schedule.js

```diff
@@ -104,23 +104,37 @@ var edx = edx || {};
 	    // Add unit handler
 	    $('#add-unit-button').on('click', function(event) {
 	      event.preventDefault();
+	      // Default value of time is 00:00.
+	      var start = self.get_datetime('start');
```

Comment:
If it's our style to use a single very large `var` block, then we should add this into that. ie: replace the `;` with a `,` and remove the var on line 109.